### ISSUE TITLE
Where does ConsoleType come from?

### ISSUE THREAD

**Issue by kaiyoma** [nils-hoyer/cypress-fail-on-console-error#7]
In your example for setting the array of console types, you have this:

```
    includeConsoleTypes: [
        ConsoleType.ERROR,
        ConsoleType.WARN,
        ConsoleType.INFO,
    ],
```

Where does `ConsoleType` come from? I tried importing it like this, but it doesn't work:

```
import failOnConsoleError, { ConsoleType } from 'cypress-fail-on-console-error';
```

Here's the error:

```
  1) An uncaught error was detected outside of a test:
     TypeError: The following error originated from your test code, not from Cypress.

  > Cannot read property 'ERROR' of undefined
```

I glanced through your source code and it doesn't look like you're exporting these values. On top of that, it's a TypeScript enum. How am I supposed to use it if my Cypress support file is written in JavaScript? It seems like using plain JavaScript strings would be better here.

**Comment by kaiyoma** [comment]
For anyone else who runs into this, you can pass numbers directly, which correspond to the enum declared here: https://github.com/nils-hoyer/cypress-fail-on-console-error/blob/9d08faac0bf761d7938311a6aea3a4bafd26eb80/src/types/ConsoleType.ts#L1

For example, this will flag warnings and errors:

```
failOnConsoleError({
  // 0 = info, 1 = warning, 2 = error
  includeConsoleTypes: [1, 2],
});
```

**Comment by nils-hoyer** [comment]
Thank you very much, for all the detailed information.
I agree with you, for supporting both TS and JS, strings be the best developer experience. 

For now I simply made ConsoleType available from the module and add your JS workaround to the readme (thanks a lot!)

fixed with version 2.0.4:
https://github.com/nils-hoyer/cypress-fail-on-console-error/releases/tag/2.0.4

commit hash: https://github.com/nils-hoyer/cypress-fail-on-console-error/commit/de096262c385c31238cbe8df85eacb9939e39ff3
